dashboard/deploy.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: v1
```

Comment:
Should we use a helm package to make it easy to configure? Should it be part of the existing helm package or have its own helm package?

Reply:
I was going to suggest integrating it as part of the current helm chart, with options to enable/disable the dashboard as well as wether it should be exposed on an internal or external IP.